roles/epel/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Check if EPEL is installed
+  stat:
+    path: /etc/yum.repos.d/epel.repo
+  register: st
```

Comment:
`st` is too generic and doesn't really describes well what is var about. How about `epel_is_installed` or similar?